tick/inference/hawkes_conditional_law.py

```diff
@@ -516,14 +564,14 @@ def incremental_fit(self, realization, compute=True):
             self._set('_lock', pool.lock)
 
         for index, (i, j, l) in enumerate(self._index2ijl):
-            if len(realization[i][0]) == 0 or len(realization[j][0]) == 0:
-                msg = "Some realizations are empty, it is not allowed"
-                raise ValueError(msg)
+#            if len(realization[i][0]) == 0 or len(realization[j][0]) == 0:
+#                msg = "Some realizations are empty, it is not allowed"
+#                raise ValueError(msg)
```

Comment:
yes, this was one of the point of the update. However if a component is empty on all realisations an error is raised when compute is called.